commands/mr/create/mr_create.go

```diff
@@ -135,6 +136,7 @@ func NewCmdCreate(f *cmdutils.Factory, runE func(opts *CreateOpts) error) *cobra
 		},
 	}
 	mrCreateCmd.Flags().BoolVarP(&opts.Autofill, "fill", "f", false, "Do not prompt for title/description and just use commit info")
+	mrCreateCmd.Flags().BoolVarP(&opts.FillCommitBody, "fill-commit-body", "", false, "Fill description with each commit body when multiple commits")
```

Comment:
More context need in the flag description.
```
	mrCreateCmd.Flags().BoolVarP(&opts.FillCommitBody, "fill-commit-body", "", false, "Fill description with each commit body when multiple commits. Can only be used with --fill")
```

And it should error if `--fill-commit-body is used without --fill`. Just like as done on line 119:

```go
if opts.FillCommitBody && !opts.Autofill {
	return &cmdutils.FlagError{Err: errors.New("--fill-commit-body should be used with --fill")}
}
```